README.md

```diff
@@ -1,7 +1,10 @@
-# Overview
+# Charmed NATS VM Operator
```

Comment:
Why `VM Operator`?

Reply:
Just copied the README template from the postgres charm. Also it makes sense since this is a machine charm?

Reply:
Lets drop the "VM" term entirely as it will lead to confusion otherwise. There is nothing which makes this a charm specific to VMs. It's for "machines", so can run on bare metal as well.